workitem/link/link_repository.go

```diff
@@ -279,3 +283,36 @@ func (r *GormWorkItemLinkRepository) Save(ctx context.Context, lt app.WorkItemLi
 	result := ConvertLinkFromModel(res)
 	return &result, nil
 }
+
+// ListWorkItemChildren get all child work items
+func (r *GormWorkItemLinkRepository) ListWorkItemChildren(ctx context.Context, parent string) ([]*app.WorkItem, error) {
+	defer goa.MeasureSince([]string{"goa", "db", "workitem", "children", "query"}, time.Now())
+	wilt := WorkItemLinkType{}
+	r.db.Where("forward_name = ?", "relates to").First(&wilt)
```

Comment:
@baijum yes, it may be hard coded but the story doesn't say anything about "relates to" as being the WILT to use. I suggest you create a new WILT to be used explicitly for this purpose because that will allow for a better db migration in the future.